README.md

```diff
@@ -1,13 +1,22 @@
 # XMODEM.rs
 
-[![Build Status](https://travis-ci.org/awelkie/xmodem.rs.svg?branch=master)](https://travis-ci.org/awelkie/xmodem.rs)
-[![](https://img.shields.io/crates/v/xmodem.svg)](https://crates.io/crates/xmodem)
-[![](https://img.shields.io/crates/l/xmodem.svg)](https://crates.io/crates/xmodem)
-
```

Comment:
Should this remain if Allen still publishes a release?
When ownership is transfered, we'd just need to renew the CI setup.

Reply:
It could do. We could probably switch over to github actions for CI.